apps/Codex/install

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+DIRECTORY="$(dirname "$(dirname "$( cd "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )")")"
+
+function error {
+  echo -e "\\e[91m$1\\e[39m"
+  exit 1
+}
+
+# Install Node.js first
+"${DIRECTORY}/manage" install-if-not-installed "Node.js" "$(dirname "$0")" || error "Failed to install Node.js"
+"${DIRECTORY}/pkg-install" "git" "$(dirname "$0")" || exit 1 
```

Comment:
No need to install git as it's installed when pi-apps is installed.

Reply:
Thanks, will remove.